test/lit/fuzz-preserve-imports-exports.wast

```diff
@@ -0,0 +1,47 @@
+;; Test the flag to preserve imports and exports in fuzzer generation.
+
+;; Generate fuzz output using this wat as initial contents, and with the flag to
+;; preserve imports and exports. There should be no new imports or exports, and
+;; old ones must stay the same.
+
+;; RUN: wasm-opt %s.ttf --initial-fuzz=%s -all -ttf --fuzz-preserve-imports-exports --metrics -S -o - | filecheck %s --check-prefix=PRESERVE
+
+;; PRESERVE: [exports]      : 1
+;; PRESERVE: [imports]      : 5
+
+;; [sic] - we do not close ("))") some imports, which have info in the wat
+;; which we do not care about.
+;; PRESERVE:  (import "a" "d" (memory $imemory
+;; PRESERVE:  (import "a" "e" (table $itable
+;; PRESERVE:  (import "a" "b" (global $iglobal i32))
+;; PRESERVE:  (import "a" "f" (func $ifunc
+;; PRESERVE:  (import "a" "c" (tag $itag
+
+;; PRESERVE:  (export "foo" (func $foo))
+
+;; And, without the flag, we do generate both imports and exports.
+
+;; RUN: wasm-opt %s.ttf --initial-fuzz=%s -all -ttf                                 --metrics -S -o - | filecheck %s --check-prefix=NORMAL
```

Comment:
I think the large space here hurts readability more than it helps. Another way to achieve the desired effect would be to put the `| filecheck ...` on a second `RUN:` line, using a backslash at the end of the first `RUN:` line to join them.

Reply:
Done.